alerta/app/database.py

```diff
@@ -1108,6 +1108,40 @@ def is_user_valid(self, id=None, name=None, login=None):
         if login:
             return bool(self._db.users.find_one({"login": login}))
 
+
+    def update_user(self, login, fields=None):
```

Comment:
This change will break compatibility with `python-alerta`, which assumes that the functions. I am generally ok with this, since the server should enforce API changes. Are you ok with it?